app/views/languages/language.erb

```diff
@@ -18,35 +18,52 @@
           </li>
         <% end %>
 
-        <li class="<%= 'active' if track.active? %>">
-          <a data-toggle="tab" href="#about">About the <%= track.language %> Track</a>
+        <li class="<%= 'active' if topic == "about" %>">
```

Comment:
It would be good to extract this to a method so that there is no conditional logic in the individual lines.
```
<li class="<%= active('about') %>">
```